packages/preferences/src/browser/views/components/preference-select-input.ts

```diff
@@ -39,6 +40,7 @@ export class PreferenceSelectInputRenderer extends PreferenceLeafNodeRenderer<JS
         const defaultValue = this.preferenceNode.preference.data.default;
         for (let i = 0; i < values.length; i++) {
             const value = `${values[i]}`;
+            const label = escapeInvisibleChars(`${values[i]}`);
```

Comment:
Suggestion: I believe this might actually be an error in the original implementation (my fault). Using the stringified version in the `deepEqual` check is incorrect (but usually works). It should probably look like this:

```suggestion
            const value = values[i];
            const stringValue = `${value}`;
            const label = escapeInvisibleChars(stringValue);
```

And later use `value: stringValue` instead. Do you mind making this change as well?

Reply:
I got it, modified.